validator/src/bin/solana-test-validator.rs

```diff
@@ -369,12 +369,13 @@ fn main() {
                 .help("deactivate this feature in genesis.")
         )
         .arg(
-            Arg::with_name("max_compute_units")
-                .long("max-compute-units")
+            Arg::with_name("max_compute_unit_limit")
```

Comment:
Why keep the "max"?

Reply:
I thought this was overriding the max compute unit limit (1.4M) that a program could use. Felt like max was appropriate here.

Reply:
Is there a limit that isn't max?

Reply:
I think `max_units` turned into `max_compute_units` so that its use here was clear.  If we take the new naming we should just keep it as `compute_unit_limit`